test/e2e/page-objects/pages/home/bitcoin-homepage.ts

```diff
@@ -0,0 +1,112 @@
+import HomePage from './homepage';
+
+class BitcoinHomepage extends HomePage {
+  protected readonly balance =
+    '[data-testid="coin-overview__primary-currency"]';
+
+  private readonly bridgeButton = {
+    text: 'Bridge',
+    tag: 'button',
+  };
+
+  private readonly buySellButton = '[data-testid="coin-overview-buy"]';
+
+  private readonly receiveButton = '[data-testid="coin-overview-receive"]';
+
+  protected readonly sendButton = '[data-testid="coin-overview-send"]';
+
+  private readonly swapButton = {
+    text: 'Swap',
+    tag: 'button',
+  };
+
+  async check_pageIsLoaded(): Promise<void> {
+    try {
+      await this.driver.waitForMultipleSelectors([
+        this.sendButton,
+        this.buySellButton,
+        this.receiveButton,
+      ]);
+    } catch (e) {
+      console.log('Timeout while waiting for bitcoin homepage to be loaded', e);
+      throw e;
+    }
+    console.log('Bitcoin homepage is loaded');
+  }
+
+  /**
+   * Checks if the bridge button is enabled on bitcoin account homepage.
+   *
+   */
+  async check_ifBridgeButtonIsEnabled(): Promise<boolean> {
+    try {
+      await this.driver.findClickableElement(this.bridgeButton, 1000);
+    } catch (e) {
+      console.log('Bridge button not enabled', e);
+      return false;
+    }
+    console.log('Bridge button is enabled');
+    return true;
+  }
+
+  /**
+   * Checks if the expected bitcoin balance is displayed on homepage.
+   *
+   * @param expectedBalance - The expected bitcoin balance to be displayed. Defaults to '0'.
+   */
+  async check_expectedBitcoinBalanceIsDisplayed(
+    expectedBalance: number = 0,
+  ): Promise<void> {
+    console.log(
+      `Check if expected bitcoin balance is displayed: ${expectedBalance} BTC`,
+    );
+    await this.driver.waitForSelector({
+      css: this.balance,
+      text: `${expectedBalance}BTC`,
+    });
+  }
+
+  /**
+   * Checks if the buy/sell button is enabled on bitcoin account homepage.
+   */
+  async check_ifBuySellButtonIsEnabled(): Promise<boolean> {
```

Comment:
just small nit, feel free to take or leave it :pray: 

```suggestion
  async check_isBuySellButtonEnabled(): Promise<boolean> {
```

Reply:
Done